recipes/_linux.rb

```diff
@@ -85,6 +85,7 @@
       node["sensu"]["version_suffix"]
     )}
     allow_downgrade true
+    flush_cache [ :before ]
```

Comment:
`flush_cache node['sensu']['yum_flush_cache'] unless node['sensu']['yum_flush_cache'].nil?` and add a default value of `nil` this allows it to be backwards compatible regardless of whether this is the right fix.